systers_portal/community/tests/selenium/test_user_actions.py

```diff
@@ -0,0 +1,60 @@
+from common.tests.selenium.base import SeleniumTestCase
+from selenium.webdriver.support.ui import Select
+
+
+class TestUserActions(SeleniumTestCase):
+    """
+    Automated visual tests for logged in user actions on community app.
+    """
+
+    def test_can_request_community(self):
```

Comment:
This is a very long test. Please split these in smaller logical sections and instead of one test, write multiple tests in this class.

Reply:
It's a single unit test and cannot be broken further... The form being tested is very long and all of it's inputs need to first be captured then secondly filled using send_keys hence the length of the method. If it was a scenario where re-usability is possible then creating another function or method that takes arguments would be an option but there's no obvious indication that we me ever want to repeat/reuse the snippets in there.